impl/maven-impl/src/main/java/org/apache/maven/impl/model/DefaultModelProcessor.java

```diff
@@ -114,9 +114,9 @@ public Model read(XmlReaderRequest request) throws IOException {
             }
             try {
                 return doRead(request);
-            } catch (IOException e) {
-                exceptions.forEach(e::addSuppressed);
-                throw e;
```

Comment:
would suggest forbidding one char vars only allowing in for loops like in cs itself.